### PR TITLE
Code of Conduct and Contributing draft

### PR DESCRIPTION
**Pull request by ken479**
First draft of code of conduct and contributing for review.

### CODE REVIEW

File: CodeofConduct.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## **Our Responsibilities**
+
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+## **Scope**
+
+This Code of Conduct applies to all content in Certifai and Classifai github, within all project spaces, and it also applies when an individual is representing Cerifai or Classifai, the project or its community in public spaces.
```
Comment (codenamewei) [inline CodeofConduct.md:31]: typo -> Cerifai

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+# **Table of content** #
+
+* [Contributing to Classifai](#contributing-to-classifai)
+* [Contribution Guildeline](#contribution-guideline)
```
Comment (codenamewei) [inline Contributing.md:4]: typo -> Guildeline

File: Contributing.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+```
+    git commit -m "message"
+```
+- Push the commit to the repository
+```
+    git push -u origin new-branch-name
+```
+- Create pull request at github
+
+You did it! Thank you so much for your contribution. We are so appreciated. Kindly wait for further notice while we checking you codes. We will get back to you soon through email or comment at github.
```
Comment (codenamewei) [inline Contributing.md:119]: either below will works
- while we are checking 
- while we check

**Review by codenamewei — CHANGES_REQUESTED** [review]

File: CodeofConduct.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Classifai Code of Conduct
+
+In the interest in fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethinicity, gender identity, and expression, level of expertise, nationality, personal appearance, race, religion, or sexual identity or orientation.
```
Comment (YCCertifai) [inline CodeofConduct.md:3]: typo for ethnicity

File: CodeofConduct.md (hunk 1 above)
Comment (ken479) [reply CodeofConduct.md:31]: @codenamewei Thank you chia wei. Will correct it.

File: Contributing.md (hunk 2 above)
Comment (ken479) [reply Contributing.md:4]: @codenamewei Thank you chia wei. Will correct it.

File: Contributing.md (hunk 3 above)
Comment (ken479) [reply Contributing.md:119]: @codenamewei Thank you chia wei. Will correct it.

File: CodeofConduct.md (hunk 4 above)
Comment (ken479) [reply CodeofConduct.md:3]: @YCCertifai Thank you. will correct it.

**Review by codenamewei — APPROVED** [review]
ok from my side

**Review by YCCertifai — APPROVED** [review]
looks good to me too

**Pull request merged**